client/__tests__/e2e/__snapshots__/e2e.test.js.snap

```diff
@@ -0,0 +1,5 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`did launch page launched 1`] = `"<span style=\\"width: 185px; display: flex; overflow: hidden; justify-content: flex-start;\\"><span style=\\"overflow: hidden; text-overflow: ellipsis; white-space: nowrap; flex-shrink: 1; min-width: 5px;\\">pbm</span><span style=\\"color: transparent; position: relative; overflow: hidden; white-space: nowrap;\\">c3k<span style=\\"position: absolute; right: 0px; color: initial;\\">c3k</span></span></span>"`;
+
+exports[`metadata loads categories and values from dataset appear 1`] = `undefined`;
```

Comment:
is this one expected to be `undefined` 😄 ? Thank you!

Reply:
Oof no, I'll have to take a look into that. I didn't even think of looking to see what was being stored in the snapshot. Thanks for catching this!

Reply:
My pleasure 🚀 ! Yeah curious to see what caused the `undefined` 😆